pkg/cliClient/cliClient_test.go

```diff
@@ -335,12 +337,26 @@ func readMock(path string) ([]extractor.Configuration, error) {
 	yamlDecoder := yaml.NewDecoder(bytes.NewReader(content))
 
 	for {
-		var doc = map[string]interface{}{}
-		err = yamlDecoder.Decode(&doc)
+		var anyTypeStore interface{}
```

Comment:
Implementing the function again is a bit weird.  Anyway, I think we can delete `readMock`